src/assets/icon/index.ts

```diff
@@ -1,3 +1,16 @@
 import IcCgvLogo from './ic_cgv_logo.svg?react';
 
 export { IcCgvLogo };
+
+import IcHamburger from './ic_hamburger_black.svg?react';
+
+export { IcHamburger };
+
+import IcSearch from './ic_search.svg?react';
+
+export { IcSearch };
+
+import IcTicket from './ic_ticket.svg?react';
+
+export { IcTicket };
```

Comment:
P5: 혹시 이렇게 주는게 편한가..? 아니면 이렇게 주는게 낫나?! 나는 import 한번에 쭈욱 해와서 export 브라켓 안에 한번에 썼거든.. 우리끼리 스타일 맞춰보는 것도 좋을거 같다!!

Reply:
P1) 이거 말하려고 했는데 한 번에 합쳐서 씁시다!!
```suggestion
import IcHamburger from './ic_hamburger_black.svg?react';
import IcSearch from './ic_search.svg?react';
import IcTicket from './ic_ticket.svg?react';


export { IcHamburger, IcSearch, IcTicket };
```

이렇게! 그래야 서로꺼 머지할 때 충돌이 덜 납니다!!